Makefile

```diff
@@ -91,6 +91,11 @@ help:
 	@echo "        Create an RPM repository with modules"
 	@echo "    fixtures/rpm-with-modules-modified"
 	@echo "        Create an RPM repository with modules but without some dependencies."
+	@echo "    fixtures/rpm-modular"
+	@echo "        Create an RPM repository with modules, module defaults and module obsoletes."
+	@echo "        All packages taken from dnf-ci-fedora-modular repository and combined with "
+	@echo "        all modular metadata from dnf-ci-thirdparty-modular and dnf-ci-obsoletes"
+	@echo "        as those metadata is intended to operate above those packages."
```

Comment:
Oh so the modular metadata is meant to go with the packages from the other repo, they're not for a separate group of packages?

Reply:
Not necessarily here, but I would like to have the URLs documented as well.

Reply:
Yes, those modular data are meant directly for the packages taken/used.

Reply:
I moved the whole message to the README file with a link to the original repository.